.github/workflows/ci.yml

```diff
@@ -81,6 +83,9 @@ jobs:
                   node-version: '18'
                   cache: ${{ steps.cache.outputs.cache-hit != 'true' && 'yarn' || '' }}
 
+            - name: Purge Nx cache of old entries
+              run: find node_modules/.cache/nx -maxdepth 1 -mtime +7 -exec rm -rf '{}' \;
```

Comment:
I must be missing something, but if you are skipping `node_modules/.cache` up on line 68, then there would be nothing to remove here?

Reply:
Ah, so the naunce is that the `Cache Nx` step is specifically managing save/restore of `node_modules/.cache/nx` - the additions on line 68 is to ensure that the `Cache node_modules` step doesn't doubly cache the Nx cache.